code/modules/mob/living/life.dm

```diff
@@ -33,44 +33,46 @@
 		return
 	if(!loc)
 		return
-	var/datum/gas_mixture/environment = loc.return_air()
 
-	if(stat != DEAD)
-		//Mutations and radiation
-		handle_mutations_and_radiation()
+	if(!IsInStasis())
 
-	if(stat != DEAD)
-		//Breathing, if applicable
-		handle_breathing(times_fired)
+		if(stat != DEAD)
+			//Mutations and radiation
+			handle_mutations_and_radiation()
 
-	handle_diseases()// DEAD check is in the proc itself; we want it to spread even if the mob is dead, but to handle its disease-y properties only if you're not.
+		if(stat != DEAD)
+			//Breathing, if applicable
+			handle_breathing(times_fired)
 
-	if (QDELETED(src)) // diseases can qdel the mob via transformations
-		return
+		handle_diseases()// DEAD check is in the proc itself; we want it to spread even if the mob is dead, but to handle its disease-y properties only if you're not.
 
-	if(stat != DEAD)
-		//Random events (vomiting etc)
-		handle_random_events()
+		if (QDELETED(src)) // diseases can qdel the mob via transformations
+			return
 
-	//Handle temperature/pressure differences between body and environment
-	if(environment)
-		handle_environment(environment)
+		if(stat != DEAD)
+			//Random events (vomiting etc)
+			handle_random_events()
 
-	handle_fire()
+		//Handle temperature/pressure differences between body and environment
+		var/datum/gas_mixture/environment = loc.return_air()
+		if(environment)
+			handle_environment(environment)
+
+		//stuff in the stomach
+		handle_stomach()
 
-	//stuff in the stomach
-	handle_stomach()
+		handle_gravity()
```

Comment:
This can kill you if gravity is too high.
https://github.com/tgstation/tgstation/blob/5b89dea70f84fe72a653c4d65fbf056206764fc3/code/modules/mob/living/life.dm#L165